### PR TITLE
Updated synthetic image creation notebook

### PR DESCRIPTION
**Pull request by ojustino**
### Description

I made some updates to the synthetic image creation notebook after a presentation. It no longer requires VPN access and the paths should work on Windows systems now.

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [ ] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [ ] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [ ] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Did the CI pass? If not, are the failures related?
- [ ] Is a change log needed? If yes, is it added to `CHANGES.rst`?
- [ ] Is a milestone set? Milestone is only currently required for PRs related to Imviz MVP.
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

File: notebooks/concepts/generate_mosviz_photometry.ipynb [hunk 1]
```diff
@@ -63,12 +66,15 @@
...
+    "from astropy.utils.data import download_file\n",
     "from astropy.wcs import WCS\n",
+    "from pathlib import Path\n",
     "from photutils.segmentation import (detect_sources, deblend_sources,\n",
     "                                    SourceCatalog)\n",
     "\n",
     "import matplotlib.pyplot as plt\n",
-    "import numpy as np"
+    "import numpy as np\n",
+    "import zipfile"
```
Comment (pllim) [inline notebooks/concepts/generate_mosviz_photometry.ipynb:77]: zipfile is from Python lib, so it should move to the first block of imports.

File: notebooks/concepts/generate_mosviz_photometry.ipynb [hunk 2]
```diff
@@ -63,12 +66,15 @@
     "from astropy.stats import (gaussian_fwhm_to_sigma,\n",
     "                           sigma_clipped_stats, sigma_clip)\n",
     "from astropy.table import Table, join\n",
+    "from astropy.utils.data import download_file\n",
     "from astropy.wcs import WCS\n",
+    "from pathlib import Path\n",
```
Comment (pllim) [inline notebooks/concepts/generate_mosviz_photometry.ipynb:71]: pathlib is from Python lib, so it should move to the first block of imports.

**Review by pllim — APPROVED** [review]
Just a minor nitpick on the imports. Otherwise, the notebook works fine without VPN. I am approving.

**Comment by ojustino** [comment]
Thanks for reviewing, @pllim. I moved the imports around as you suggested.

**Pull request merged**